src/table/table.scss

```diff
@@ -16,75 +15,205 @@
     width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
+  }
+
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
+
+    > .iui-borderless {
+      margin-left: $iui-xs;
+
+      &:not(.iui-active) {
+        visibility: hidden;
+      }
+    }
+
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
+        margin: 0 auto;
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+        @include themed {
+          background-color: t(iui-color-background-5);
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
+      }
+    }
+  }
+
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
     @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
+      fill: t(iui-icons-color-actionable);
     }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+
+    .iui-sort {
+      visibility: visible;
+        
+      &,
+      > * {
+        @include themed {
+          fill: t(iui-icons-color-actionable);
+        }
+      }
+    }
+  }
+
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
     }
+  }
+}
+
+@mixin iui-table-body {
+  width: 100%;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    width: 100%;
+    margin-top: -1px;
+    border: solid 1px transparent;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
+    // Last row doesn't get bottom border. Up for debate.
+    &:not(:last-child) {
       @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
+        border-bottom: solid 1px t(iui-color-background-4);
+      }
+    }
+
+    > .iui-slot > .iui-more-options {
+      visibility: hidden;
```

Comment:
Do we really want to controll such details??
That just complicates implementation and not sure, what it gives in this situation...

Reply:
Without that CSS the More Options menu would appear on every row.  It adds a lot of repetitive, extra visual noise. 
![Screen Shot 2021-06-01 at 3 20 20 PM](https://user-images.githubusercontent.com/849817/120378807-e382c280-c2ec-11eb-8eb9-cbb5e94dedc7.png)

I'd prefer to only show the More Options menu on hover or focus-within the row, similar to how we show the More Options menu on the tile component.

Reply:
yeah, I understood it.. just not a fan of this... but you know better, how it should be.